webui/react/src/globalStorage.ts

```diff
@@ -28,8 +28,8 @@ class GlobalStorage {
     this.storage.set(this.keys.serverAddress, address);
   }
 
-  get getServerAddress() {
-    return this.storage.get<string>(this.keys.serverAddress);
+  get serverAddress() {
+    return this.storage.get<string>(this.keys.serverAddress) || '';
```

Comment:
👍  I wasn't sure about the ` || ''` and surprised to see the following test results:
```
const abc = '' || 'abc';
// abc evaluates to be 'abc'
```